Jupyter notebook for the smartAS project. The aim of the project is the identification of alternative splicing alterations with potential to act as tumor drivers.

# Analyses

## Manuscript figures and tables

* [Manuscript figures](switches/figures.ipynb)
* [Manuscript supplementary figures](switches/sup_figures.ipynb)
* [Manuscript supplementary files](switches/sup_files.ipynb)

## Switches

* [Switch statistics](switches/switches_stats.ipynb)
* [Immune cells/stroma contamination](switches/immune_analysis.ipynb)
* [Functionality](switches/functionals.ipynb)
* [Tumorigenic evidences](switches/potus.ipynb)

## Mutations

* [Whole-exome sequencing gene analysis](mutations/wes_genes.ipynb)
* [Whole-genome sequencing gene analysis](mutations/wgs_genes.ipynb)
* [Mutual exclusion: comet](mutations/me_comet.ipynb)

## Structure

* [Pfam domains](structural_analysis/pfam_switches.ipynb)
* [Protein-protein interactions](structural_analysis/ppi.ipynb)
* [Protein-protein interactions stats](structural_analysis/ppi_stats.ipynb)
* [Protein-protein interactions mutual exclusion](structural_analysis/ppi_me.ipynb)
* [Protein-protein interactions neighborhoods](structural_analysis/ppi_neighborhoods.ipynb)
* [Neighborhood curation](structural_analysis/ppi_interesting_neighborhoods.ipynb)

# Data

The data was generated by applying the [SmartAS pipeline](https://bitbucket.org/regulatorygenomicsupf/smartas) to transcript expression data generated by the TCGA project. The description of each file can be found [here](data/README.md).
